docs/routeCurrent.md

```diff
@@ -1,22 +1,47 @@
 @function can-stache-route-helpers.routeCurrent {{# routeCurrent(hash) }}
 @parent can-stache-route-helpers
 
-Returns if the hash values match the [can-route]’s current properties.
+@description Returns if the hash values match the [can-route]’s current properties.
 
 @signature `routeCurrent( hashes... [,subsetMatch] )`
 
-  Calls [can-route.isCurrent route.isCurrent] with `hashes` and returns the result. This
-  can be used in conjunction with other helpers:
-
-```html
-{{linkTo("Todos", routeCurrent(page='todos' id=todo.id))}}
-```
-
-Or on its own:
-
-```html
-<a class="{{# routeCurrent(page='todos', true) }}active{{/ routeCurrent }}">Todos</a>
-```
+  Calls [can-route.isCurrent route.isCurrent] with `hashes` and returns the result. The next example adds the `'active'` class on the anchor if `page` is equal to `todos`.
+
+  ```html
+  <mock-url></mock-url>
+  <cooking-example></cooking-example>
+  <style>
+  .active { background-color: darkred; }
+  </style>
+  <script type="module">
+  import {Component} from "can";
+  import "//unpkg.com/mock-url@^5";
+  Component.extend({
+    tag: "cooking-example",
+    view: `<li {{# routeCurrent(page="recipe" id=5) }}class='active'{{/ routeCurrent }}>
+        <a href='{{ routeUrl(page="recipe" id=5) }}'>{{recipe.name}}</a>
+      </li>`,
+    ViewModel: {
+      recipe: {
+        default() {
+          return {name: "apple pie"};
+        }
+      }
+    }
+  });
+  </script>
+  ```
+  @codepen
+
+  With default routes and a url like `#!&page=recipe&id=5`, this produces:
+
+  ```html
+  <li class='active'>
+    <a href='#!&page=recipe&id=5'>{{recipe.name}}</a>
+  </li>
+  ```
+
+  `routeCurrent` can also be used with other [can-stache.Helpers].
```

Comment:
probably need an example here or remove.  Also, I'd link to `[can-stache stache helpers]` because `can-stache.Helpers` is being removed.

Reply:
```
disabled:from="or( routeCurrent( page="admin" ), not( user.canAccess( page="admin" ) )"
```